components/split-view/listbox.jsx

```diff
@@ -0,0 +1,383 @@
+	 * * * `descending`: Descending sorting.
+	 * * * `ascending`: Ascending sorting.
+	 */
+	assistiveText: PropTypes.shape({
+		list: PropTypes.string,
+		sort: PropTypes.shape({
+			sortedBy: PropTypes.string,
+			descending: PropTypes.string,
+			ascending: PropTypes.string
+		}),
+		unreadItem: PropTypes.string
+	}),
+	/**
+	 * CSS classes to be added to the parent `div` tag.
+	 */
+	className: PropTypes.oneOfType([
+		PropTypes.array,
+		PropTypes.object,
+		PropTypes.string
+	]),
+	/**
+	 * Event Callbacks
+	 * * `onSelect`: Called when a list item is selected.
+	 * * * event {object} List item click event
+	 * * * Meta {object}
+	 * * * * selectedItems {array} List of selected items.
+	 * * * * item {object} Last selected item.
+	 * * `onSort`: Called when the list is sorted.
+	 * * * event {object} Sort click event
+	 */
+	events: PropTypes.shape({
+		onSelect: PropTypes.func.isRequired,
+		onSort: PropTypes.func
+	}),
+	/**
+	 * HTML id for component.
+	 */
+	id: PropTypes.string,
+	/**
+	 * **Text labels for internationalization**
+	 * * `header`: This is the header of the list.
+	 */
+	labels: PropTypes.shape({
+		header: PropTypes.string
+	}),
+	/**
+	 * The direction of the sort arrow. Option are:
+	 * * SORT_OPTIONS.UP: `up`
+	 * * SORT_OPTIONS.DOWN: `down`
+	 */
+	sortDirection: PropTypes.oneOf([SORT_OPTIONS.UP, SORT_OPTIONS.DOWN]),
+	/**
+	 * Allows multiple item to be selection
+	 */
+	multiple: PropTypes.bool,
+	/**
+	 * The list of items.
+	 * It is recommended that you have a unique `id` for each item.
+	 */
+	options: PropTypes.array.isRequired,
+	/**
+	 * Accepts an array of item objects. For single selection, pass in an array of one object.
+	 */
+	selection: PropTypes.array,
+	/**
+	 * Accepts an array of item objects. For single unread, pass in an array of one object.
+	 */
+	unread: PropTypes.array,
+	/**
+	 * Custom list item template for the list item content. The select and unread functionality wraps the custom list item.
+	 * This should be a React component that accepts props.
+	 */
+	listItem: PropTypes.func
+};
+
+const defaultProps = {
+	assistiveText: {
+		list: 'Select an item to open it in a new workspace tab.',
+		sort: {
+			sortedBy: 'Sorted by',
+			descending: 'Descending',
+			ascending: 'Ascending'
+		}
+	},
+	events: {},
+	labels: {},
+	selection: [],
+	unread: []
+};
+
+class SplitViewListbox extends React.Component {
+	static displayName = SPLIT_VIEW_LISTBOX;
+	static propTypes = propTypes;
+	static defaultProps = defaultProps;
+
+	constructor (props) {
+		super(props);
+
+		this.listItemComponents = {};
+
+		this.state = {
+			currentSelectedItem: null,
+			currentFocusedListItem: {
+				index: 0,
+				item: null
+			}
+		};
+	}
+
+	componentWillMount () {
+		// Generates the list item template
+		this.ListItemWithContent = listItemWithContent(
+			this.props.listItem || SplitViewListItemContent
+		);
+	}
+
+	componentDidMount () {
+		this.focusFirstItem();
```

Comment:
It doesn't actually steal focus. It just sets the `currentFocusedListItem` state so that when you tab into the listbox for the first time the focus moves to the correct list item.